src/main/java/org/example/Main.java

```diff
@@ -28,14 +33,30 @@ public static void main(String[] args) throws IOException {
         serializer.configure(config, false);
 
         var avroRecord = AvroUtils.jsonToGenericRecord(eventData, avroSchema);
-        byte[] avroBytes = serializer.serialize(topic, avroRecord);
-
-        System.out.println("Encoded bytes: " + Arrays.toString(avroBytes));
+//        byte[] avroBytes = serializer.serialize(topic, avroRecord);
 
+//        System.out.println("Encoded bytes: " + Arrays.toString(avroBytes));
+//
         var deserializer = new KafkaAvroDeserializer();
         deserializer.configure(config, false);
-        var deserialized = deserializer.deserialize(topic, avroBytes);
+//        var deserialized = deserializer.deserialize(topic, avroBytes);
+
+//        System.out.println("Deserialized Message: " + deserialized.toString());
+
+        var debezium = new JsonSchemaData();
+        var json = AvroUtils.readJson(eventData);
+        var payloadSchema = debezium.toConnectSchema("event", json);
+        var payload = debezium.toConnectData(json, payloadSchema);
+        System.out.println("Connect Schema from Debezium: " + payloadSchema.toString());
```

Comment:
```suggestion
        System.out.println("Connect Schema from Debezium: " + ToStringBuilder.reflectionToString(payloadSchema));
```

Using the above yields:
```
Connect Schema from Debezium: org.apache.kafka.connect.data.ConnectSchema@193f604a[defaultValue=<null>,doc=<null>,fields=[Field{name=foo, index=0, schema=Schema{STRING}}],fieldsByName={foo=Field{name=foo, index=0, schema=Schema{STRING}}},hash=2145950518,keySchema=<null>,name=event,optional=true,parameters=<null>,type=STRUCT,valueSchema=<null>,version=<null>]
Connect Data from Debezium: Struct{foo=123}
```

Reply:
Using the above yields:
```
Connect Schema from Debezium: org.apache.kafka.connect.data.ConnectSchema@193f604a[defaultValue=<null>,doc=<null>,fields=[Field{name=foo, index=0, schema=Schema{STRING}}],fieldsByName={foo=Field{name=foo, index=0, schema=Schema{STRING}}},hash=2145950518,keySchema=<null>,name=event,optional=true,parameters=<null>,type=STRUCT,valueSchema=<null>,version=<null>]
Connect Data from Debezium: Struct{foo=123}
```

Reply:
This shows that the schema is coming over correctly from Debezium which points to the bug being inside of the AvroConverter